tests-functional/clients/anvil.py

```diff
@@ -0,0 +1,44 @@
+import logging
+import time
+import docker
+
+from conftest import option
+from web3 import Web3
+
+
+class Anvil(Web3):
```

Comment:
I guess it makes sense to have separate Docker class to be used both in Anvil/StatusBackend classes 
not for this PR tho